ui/apps/platform/src/reducers/policies/table.js

```diff
@@ -1,13 +1,8 @@
-import { combineReducers } from 'redux';
-
 // Action types
 //-------------
 
 export const types = {
     UPDATE_POLICY_DISABLED_STATE: 'policies/UPDATE_POLICY_DISABLED_STATE',
```

Comment:
it looks like update policy disabled state is being used in vuln mgmt so i left it as is for now.

Reply:
The e2e test that's failing in CI looks like it may be related to this, so maybe there's something else we need to retain from the old code.

Reply:
Failing integration test is because of classic selector for PatternFly criteria element